assets/javascripts/discourse/lib/multilingual-category.js.es6

```diff
@@ -16,6 +16,15 @@ function translatedCategoryName(category) {
   return translatedName || get(category, "name");
 }
 
+function translatedCategoryDescription(category) {
```

Comment:
@merefield I feel like this logic could be simplified. It feels a bit spaghetti.

Reply:
As discussed, this is to protect against the attribute being missing.